### PR TITLE
Update/iteration3 data model

### PR DESCRIPTION
**Pull request by jennadilullo8**
**Update Data Model**

- Driver: Jenna
- Navigator : Alex, Liliana

- Updated data model with star button being true.
- Fixed syntax errors and implemented dryer code.
- Refactored global variables to be local variables, mainly in the show starred button function.

- Attempted to refactor the display of the show filtered ideas and starred button after the menu close button is clicked.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -80,22 +80,32 @@ function clearInput() {
...
-    }
+      deleteIdea(event);
+  }
 }
 
 function deleteIdea(event) {
   var uniqueID = event.target.closest('.x-button').dataset.id;
   for (var i = 0; i < ideaArray.length; i++) {
-    if (uniqueID == ideaArray[i].id) {
+    if (uniqueID === ideaArray[i].id) {
```
Comment (lilianaweimer) [inline src/main.js:90]: Good call changing == to ===!

**Review by lilianaweimer — APPROVED** [review]
Looks good!

File: src/main.js [hunk 2]
```diff
@@ -27,36 +21,42 @@ ideaCardsSection.addEventListener('click', function() {
 });
 
 //event handlers
-function showFilterStarred() {
+function showFilterStarred(event) {
```
Comment (asodemann18) [inline src/main.js:24]: Nice job of switching from a global to local variable.

**Review by asodemann18 — COMMENTED** [review]
Good job refactoring and getting the favoriting functionality to work!

**Pull request merged**